File: Java programs/Problems on Pattern Printing/Programs to print Alphabetical Square pattern/Pattern1.java
/* Write a java program which Accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=4   iCol=4
Output : A B C D
         A B C D
         A B C D
         A B C D  
         
*/
		  
import java.util.*;

class Pattern
{
	public String str;
    public void Pattern(int iRow,int iCol)
    {
	   int i=0;
	   int j=0;
	   char ch='\0';
	   for(i=1;i<=iRow;i++)
	   {  
           for(j=1,ch='A';j<=iCol;j++,ch++)
	       {
			   System.out.print(ch+"\t");
		   }
	      System.out.println();
        }
	}
}

class Lba361
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int iValue1=0,iValue2=0;

        System.out.println("Enter number of Rows ");
        iValue1=sobj.nextInt();

		 System.out.println("Enter number of Columns");
		iValue2=sobj.nextInt();

		Pattern pobj=new Pattern();

        pobj.Pattern(iValue1,iValue2);
	}	
}  	  

File: C  programs/Problems on Pattern Printing/Triangle Pattern/Pattern1.c
/* Accept the number of rows and number of columns from user  and Display below pattern.

Input:  iRow=4  iCol=4
Output : 
         * 
         * * 
         * * * 
	     * * * *
*/
		  
#include<stdio.h>

void Pattern(int iRow,int iCol)
{
	int i=0,j=0;

    if(iRow<0)
	{
	   iRow=-iRow;
	}

	if(iCol<0)
	{
	   iCol=-iCol;
	}

	if(iRow !=iCol)
	{
	  printf("Invalid Input");
	  return;
	}  

	for(i=1;i<=iRow;i++)  
    {
	   for(j=1;j<=iCol;j++)  
       {
		  if(i>=j)        
          {
			 printf("*\t");
	      }
		  
	   }
	   printf("\n");
	}   
}

int main()
{
  int iValue1=0,iValue2=0;

  printf("Enter number of rows and columns");
  scanf("%d %d",&iValue1,&iValue2);

  Pattern(iValue1,iValue2);

  return 0;
 } 